packages/cli/src/utils.ts

```diff
@@ -259,11 +246,7 @@ export async function readProjectSecrets(
   }
   const secretFile = secretFileResult.value;
   if (!fs.existsSync(secretFile)) {
-    if (isMultiEnvEnabled()) {
```

Comment:
No problem with removing this check  to multi-env?

Reply:
No problem because `isMultiEnvEnabled` is always `true` in current version.

Reply:
got it.